aineko/cli/dream.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""A wrapper around API calls to the Aineko Dream API."""
+
+import click
+import requests
+
+
+@click.command()
+@click.argument("prompt")
+@click.argument("api-key")
+@click.option(
+    "-u",
+    "--url",
+    default="https://dream.ap-2359264f-9cba-43ba-9ac4-ae3ba7b1d4b1.aineko.app",
```

Comment:
Should we make this a config? As we continue to shuffle some backend infrastructure this Project ID might change.